Makefile

```diff
@@ -46,16 +46,19 @@ requirements: node_requirements python_requirements ## install development envir
 install_linux_dev_firefox: ## Downloads custom version of firefox for Selenium in Linux
 	@echo "This works only on Linux. For MacOS please check the README file"
 
-	rm -rf .firefox
+	rm -rf .firefox && rm -rf .geckodriver
```

Comment:
Nit: `rm` accepts multiple arguments, so `rm -rf .firefox .geckodriver` should also work.

Reply:
Done